scripts/prepare_sector_network.py

```diff
@@ -2154,6 +2164,31 @@ def add_industry(n, costs):
     else:
         shipping_bus = nodes + " H2"
 
+    if options.get("shipping_methanol"):
+
+        n.madd("Bus",
+            spatial.methanol.nodes,
+            carrier="methanol",
+            location=spatial.methanol.locations,
+            unit="MWh_LHV"
+        )
+
+        # methanolisation
+        n.madd(
+            spatial.h2.locations + "methanolisation",
+            bus0=spatial.h2.nodes,
+            bus1=spatial.methanol.nodes,
+            bus2=nodes,
+            bus3=spatial.co2.nodes,
+            carrier="methanolisation",
+            p_nom_extendable=True,
+            capital_cost=costs.at["methanolisation", 'fixed'] * options["MWh_MeOH_per_MWh_H2"], # EUR/MW_H2/a
+            lifetime=costs.at["methanolisation", 'lifetime'],
+            efficiency=options["MWh_MeOH_per_MWh_H2"],
+            efficiency2=- options["MWh_MeOH_per_MWh_H2"] / options["MWh_MeOH_per_tCO2"],
+            efficiency3=- options["MWh_MeOH_per_MWh_H2"] / options["MWh_MeOH_per_MWh_e"],
```

Comment:
`bus2` is electricity and `bus3` CO2, right?

Shouldn't then be `efficiency2` about MWh_e and `efficiency3` about CO2 (currently it is the other way around)

Reply:
Yup, great catch!